migrations/2018_01_30_222900_change_users_b8_columns.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Schema\Builder;
+
+return [
+    'up' => function (Builder $schema) {
+        $schema->table('users', function (Blueprint $table) {
+            $table->renameColumn('is_activated', 'is_email_confirmed');
+            $table->renameColumn('join_time', 'joined_at');
+            $table->renameColumn('last_seen_time', 'last_seen_at');
+            $table->renameColumn('discussions_count', 'discussion_count');
+            $table->renameColumn('comments_count', 'comment_count');
+            $table->renameColumn('read_time', 'marked_all_as_read_at');
+            $table->renameColumn('notifications_read_time', 'read_notifications_at');
+            $table->renameColumn('avatar_path', 'avatar_url');
+            $table->dropColumn('bio');
```

Comment:
Let's not drop bio just yet, since we don't have a replacement yet. See https://github.com/flarum/core/issues/1280